README.md

```diff
@@ -20,20 +20,17 @@ Add to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-avro-rs = "^0.6"
+avro-rs = "0.6"
+failure = "0.1.5"
```

Comment:
Why are these needed? (failure & serde) Won't they get pulled in alongside avro-rs when it's added as a dependency?

Reply:
Heh, just saw #74 . Resolving this